app.js

```diff
@@ -27,14 +41,24 @@ app.get('/leaders', function(request, response) {
 
 io.on('connection', function(socket){
 
+  io.emit('question', getQuestion());
```

Comment:
This is the same problem that was before where I asked you to compare 3 different pieces of code.

Go  back to that question and try to think answer again.